Cargo.toml

```diff
@@ -8,6 +8,12 @@ repository = "https://github.com/sgrif/pq-sys"
 links = "pq"
 build = "build.rs"
 
+[dependencies]
+openssl-sys = { version = "*", optional = true }
```

Comment:
Is this a wildcard for flexibility? I have the impression that you can't cargo publish the crate if it uses wildcards.

Reply:
Yeah, it's to not mess with the version whoever depends on this crate chooses. I have no idea of the requirements on publishing crates so that might be true.

Maybe ^0 or ^0.9 would work instead?

Reply:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#comparison-requirements I think this is the way to go. `"version = ">= 0.9, < 0.11"`. Since version 0.11 and beyond don't exist at the moment, we can't say anything about the compatibility, so saying "up to 0.10.x" is the best we can do, I think.

Reply:
I had to go with `version = ">= 0.9.0, < 0.11"` for it to compile